static/scripts/rudderTrack.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { rudderInitialize } from "./rudderInitialize";
+
+export async function onRouteDidUpdate({location, previousLocation}) {
+  // Don't execute if we are still on the same page; the lifecycle may be fired
+  // because the hash changes (e.g. when navigating between headings)
+  if (location.pathname !== previousLocation?.pathname) {
+    await rudderInitialize()
```

Comment:
This will initialize rudderstack on each navigation. I don't think we want that.

Reply:
Yes I felt bad about this, but not sure how to avoid it?